exportid-explainer.md

```diff
@@ -1,3 +1,6 @@
+> [!CAUTION]  
+> The Export ID proposal is no longer planned. It has been replaced by the [Reference Target](reference-target-explainer.md) proposal.
+
```

Comment:
Adding this warning to the exportid explainer instead of deleting it, since (a) it's been linked from other places and can serve as a redirect to the Reference Target explainer, and (b) some ideas from it may still be useful to reference as part of the discussion on referenceTargetMap.

Reply:
But let me know if there's another preferred way to do this for the AOM repo!